source/blog/2019-01-04-the-ember-times-issue-79.md

```diff
@@ -55,7 +55,11 @@ A **new RFC** proposes the [deprecation of these abundant properties](https://gi
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [API Search Performance](#api-search-performance)
```

Comment:
Currently we try to link to a relevant article or repository for further reading in each section title. Is there a blog post or a tweet about the performance improvement you'd like to link to?
Otherwise I think linking to the client library Github would also be great, e.g.

```suggestion
## [API Search Performance](https://github.com/algolia/algoliasearch-client-javascript)
```

Reply:
ah nice suggestion! I'll do a tweet and link to it :)

Reply:
nice suggestion, added a link to my tweet!